src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7206,7 +7206,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Types and aliases cannot be named 'scoped'.</value>
   </data>
   <data name="ERR_UnscopedRefAttributeUnsupportedTarget" xml:space="preserve">
-    <value>UnscopedRefAttribute can only be applied to 'out' parameters, 'ref' parameters that refer to 'ref struct' types, and instance methods and properties on 'struct' types other than constructors and 'init' accessors.</value>
+    <value>UnscopedRefAttribute can only be applied to 'out' parameters, 'ref' and 'in' parameters that refer to 'ref struct' types, and instance methods and properties on 'struct' types other than constructors and 'init' accessors.</value>
```

Comment:
Doesn't need to happen here but it might be good to adjust this language to have distinct messages like
- *cannot use UnscopedRefAttribute here because the associated by-reference parameter is unscoped by default.*
- *cannot use UnscopedRefAttribute here because the associated parameter is by-value.*

It feels like it will be easier to spell out the precise enumeration of which parameters are unscoped by default, etc. in documentation rather than in a single diagnostic message.